src/main/kotlin/racingcar/ui/Player.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar.ui
+
+import camp.nextstep.edu.missionutils.Console
+
+class Player {
+    fun inputCarNames(): List<String> {
+        println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+        return inputWithNullCheck().split(",")
+    }
+
+    fun inputTryCount(): Int {
+        println("시도할 횟수는 몇 회인가요?")
+        val tryCount = inputIntWithNullCheck()
+        if (tryCount < 1) {
+            throw IllegalArgumentException("시도 횟수는 1 이상이어야 합니다.")
+        }
```

Comment:
다른 파일에서 오류 처리 하셨던 것 처럼 [require()](https://seosh817.tistory.com/155)로 변경해서 간결하게 수정하면 좋을 것 같아요
```suggestion
        require(tryCount >= 1) { "시도 횟수는 1 이상이어야 합니다." }
```

Reply:
그렇네요! require로 변경한다면 간결함과 코드의 일관성을 가져갈 수 있을 것 같네요 ㅎㅎ 감사합니다!